IBF-Typhoon-model/post_test.py

```diff
@@ -0,0 +1,249 @@
+
+
+    # leadtime
+    leadtime = df_landfall['time_for_landfall'].item()
+    print(leadtime)
+
+
+    # check alert threshold
+    trigger = check_trigger(df_trigger, Activetyphoon)
+    layer = 'alert_threshold'
+    df_impact[layer] = np.where(df_impact['impact']>0, 1, 0) # mark 1 when impact>0
+    admin_level = 0
+
+
+    # rainfall
+    layer = 'rainfall'
+    admin_level = 3
+    df_rain_exposure_place_codes = []
+    for ix, row in df_rain.iterrows():
+        exposure_entry = {'placeCode': row['Mun_Code'],
+                          'amount': round(row['max_24h_rain'], 2)}
+        df_rain_exposure_place_codes.append(exposure_entry)
+    to_api(admin_level, layer, df_rain_exposure_place_codes, leadtime)
+
+
+    # windspeed
+    layer = 'windspeed'
+    admin_level = 3
+    df_wind_exposure_place_codes = []
+    for ix, row in df_wind.iterrows():
+        exposure_entry = {'placeCode': row['adm3_pcode'],
+                          'amount': round(row['v_max'], 2)}
+        df_wind_exposure_place_codes.append(exposure_entry)
+    to_api(admin_level, layer, df_wind_exposure_place_codes, leadtime)
+
+    # landfall location
+    layer = 'landfall'
+    admin_level = 0
+    df_wind_exposure_place_codes = []
+    for ix, row in df_landfall.iterrows():
+        exposure_entry = {'lat': row['landfall_point_lat'],
+                          'lon': row['landfall_point_lon']}
+        df_wind_exposure_place_codes.append(exposure_entry)
+    to_api(admin_level, layer, df_wind_exposure_place_codes, leadtime)
+    
+    
+    # impact
+    layer = 'house_affected'
+    admin_level = 3
+    df_impact_exposure_place_codes = []
+    for ix, row in df_impact.iterrows():
+        exposure_entry = {'placeCode': row['adm3_pcode'],
+                          'amount': round(row['impact'], 2)}
+        df_impact_exposure_place_codes.append(exposure_entry)
+    to_api(admin_level, layer, df_impact_exposure_place_codes, leadtime)
+    
+
+    # probability within 50 km
+    layer = 'prob_within_50km'
+    admin_level = 3
+    df_impact_exposure_place_codes = []
+    for ix, row in df_impact.iterrows():
+        exposure_entry = {'placeCode': row['adm3_pcode'],
+                          'amount': round(row['probability_dist50'], 2)}
+        df_impact_exposure_place_codes.append(exposure_entry)
+    to_api(admin_level, layer, df_impact_exposure_place_codes, leadtime)
+
+
+    # alert layer
+    layer = 'alert_threshold'
+    admin_level = 3
+    df_impact_exposure_place_codes = []
+    for ix, row in df_impact.iterrows():
+        exposure_entry = {'placeCode': row['adm3_pcode'],
+                          'amount': row[layer]}
+        df_impact_exposure_place_codes.append(exposure_entry)
+    to_api(admin_level, layer, df_impact_exposure_place_codes, leadtime)
+
+    logging.info('post_output: done')
+
+
+    # landfall time
+
+
+    # landfall location
+
+
+
+
+def check_trigger(df_trigger, Activetyphoon):
```

Comment:
again no need for a function here just add a line to the output of the trigger function 
"%>%dplyr::mutate(trigger = ifelse(100k >= 50,1,ifelse(80k >= 60,1,ifelse(70k >= 50,1, ifelse(50k >= 80,1,ifelse(30k >= 95,1, 0)))))" 1 is trigger and 0 is not triggered
will implement this